internal/acceptance/volumes_test.go

```diff
@@ -0,0 +1,42 @@
+package acceptance
+
+import (
+	"testing"
+)
+
+func TestAccVolumesResourceFullLifecycle(t *testing.T) {
+	t.Skip("Not running until we have UC Volumes enabled in our aws test workspace")
```

Comment:
We have made a request for this, will check for full lifecycle once we have this capability in our workspace. Skipping for now.